lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs

```diff
@@ -249,3 +255,86 @@ spec = describe "SHELLEY_ADDRESSES" $ do
         let str = "patate"
         r <- request @Aeson.Value ctx (Link.inspectAddress str) Default Empty
         expectResponseCode @IO HTTP.status400 r
+
+    -- Generating golden test data for enterprise addresses:
+    --- $ cardano-address script hash "$(cat script.txt)" \
+    --- | cardano-address address payment --from-script --network-tag mainnet
+    it "ANY_ADDRESS_POST_01 - Golden tests for enterprise script address - signature" $ \ctx -> do
```

Comment:
Nitpicking but why golden? Afaik, golden are unit tests with output in separate file which is not the case here.

Reply:
golden because I have generated golden data in cardano-address for all examples, and check if all json/handling/.. machinery comes to the same results